pai-management/doc/how-to-write-pai-configuration.md

```diff
@@ -91,35 +91,34 @@ machine-list:
       pai-worker: "true"
 ```
 
-
-
 - ```hostname```: Required. You could the hostname by the command ```echo `hostname` ``` on the host.
 - ```hostip```: Required. The ip address of the corresponding host.
 - ```machine-type```: Required. The sku name defined in the ```machine-sku```.
 - ```etcdid```: K8s-Master Required. The etcd is part of kuberentes master. If you assign the k8s-role=master to a node, you should set this filed. This value will be used when starting and fixing k8s.
-- ```sshport, username, password```: Optional. If this machine's account and port is different from the default properties, you should fill these 3 fields. Or you can remove them.
-- ```k8s-role```: Required. You could set this value to ```master```, ```worker``` or ```proxy```. This field is relative to kubernetes ha. if you want to configure more then 1 k8s-master, please refer to this [chapter]((#k8s-high-availability-configuration)).
+- ```sshport, username, password```: Optional. Used if this machine's account and port is different from the default properties. Or you can remove them.
+- ```k8s-role```: Required. You could set this value to ```master```, ```worker``` or ```proxy```. If you want to configure more than 1 k8s-master, please refer to [Kubernetes High Availability Configuration](#k8s-high-availability-configuration).
 - ```dashboard```: Select one node to set this field. And set the value as ``` "true" ```.
-- ```zkid```: Pai-Master Required. You can set this filed from ```1``` to ```n```
 - ```pai-master```: Optional. hadoop-name-node, hadoop-resource-manager, frameworklauncher, restserver, webportal, grafana, prometheus and node-exporter.
-- ```pai-worker```: Optional. hadoop-data-node, hadoop-node-manager, node-exporter.
-- ```node-exporter```: Optional. Some machine may not have ```pai-master``` and ```pai-worker``` labels. You should assign this label to the node.
+- ```zkid```: Unique zookeeper id. Required by```pai-Master```. You can set this filed from ```1``` to ```n```
+- ```pai-worker```: Optional. hadoop-data-node, hadoop-node-manager, and node-exporter will be deployed on a pai-work.
+- ```node-exporter```: Optional. Some machine may not have ```pai-master``` and ```pai-worker``` labels. You can assign this label to the node.
```

Comment:
node-exporter has nothing to do with pai-master or pai-worker. So I suggest we don’t mention these two labels at all here to avoid confusion.